src/sqliteformat.cpp

```diff
@@ -621,11 +621,12 @@ bool SqliteFormat::modifyComponents(const Incidence::Ptr &incidence, const QStri
         }
         SL3_bind_int(stmt1, index, percentComplete);
         SL3_bind_date_time(this, stmt1, index, effectiveDtCompleted, incidence->allDay());
-        SL3_bind_int(stmt1, index, incidence->thisAndFuture());
 
         colorstr = incidence->color().toUtf8();
         SL3_bind_text(stmt1, index, colorstr.constData(), colorstr.length(), SQLITE_STATIC);
```

Comment:
Suppose this is the "extra1". Not related to these changes but would be nice to get such things renamed to what they are really used for.

Reply:
Yes, I agree, in the meantime, I've added a comment back when pushing this change in sqliteformat.h#308 to specify that extra1 is now used to store the incidence colour.

If I don't get burn too much with the current migration, I may create another one renaming the column. I agree that it would make the code less obscure.

Reply:
Hm, not spotting such comment, nor changes in the code.

Reply:
No wait, that was referring to the existing code.

Reply:
Indeed ; )